src/combinators/converge.spec.js

```diff
@@ -0,0 +1,56 @@
+const test = require('tape')
+const helpers = require('../test/helpers')
+
+const bindFunc = helpers.bindFunc
+const isFunction = require('../predicates/isFunction')
+
+const sub = require('./converge')
+
+test('converge (Big Phi or S\' combinator)', t => {
+  const s = bindFunc(sub)
+  const x = 67
+  const f = x => y => x + y
+  const g = x => x - 1
+  const h = x => x + 1
+
+  t.ok(isFunction(sub), 'is a function')
+
+  t.throws(s(undefined, g, h, x), TypeError, 'throws with first arg undefined')
```

Comment:
Another thing in the specs, typically we want to test for the specific Error Message that we provide so you could do
```js
const err = /converge: Functions required for first three arguments$/
```

and do a search and replace for `:%s/TypeError/err/g`